apis/v1alpha1/addtoscheme_integreatly_v1alpha1.go

```diff
@@ -2,6 +2,7 @@ package v1alpha1
 
 import (
 	upsv1alpha1 "github.com/aerogear/unifiedpush-operator/pkg/apis/push/v1alpha1"
+	oo "github.com/bf2fc6cc711aee1a0c2a/observability-operator/v3/api/v1"
```

Comment:
I think we should stick to the longer names `observability-operator` for variable names for code readability,  someone new to the code base wouldn't know what `oo` means

Reply:
Done